salt/journal/config/etc-nginx-traits.d-redirect-existing-paths.conf

```diff
@@ -1446,7 +1446,6 @@ map $uri $new_uri {
     '/privacy-notice' '/privacy';
     '/privacy-policy' '/privacy';
     '/terms-and-conditions-of-use' '/terms';
```

Comment:
it may be worth documenting somewhere (top of this file? README?) that URLs are not supported as destinations here, only paths